src/applications/vaos/components/cancel/CancelCernerAppointmentModal.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Modal from '@department-of-veterans-affairs/formation-react/Modal';
+import { FETCH_STATUS } from '../../utils/constants';
+import LoadingButton from 'platform/site-wide/loading-button/LoadingButton';
+
+export default function CancelCernerAppointmentModal({
+  onClose,
+  onConfirm,
+  status,
+}) {
+  return (
+    <Modal
+      id="cancelCernerAppt"
+      status="warning"
+      visible
+      onClose={onClose}
+      title="You cannot cancel this appointment here."
+    >
+      To cancel this appointment, sign in on My VA health.
+      <p className="vads-u-margin-top--2">
+        <LoadingButton
```

Comment:
I think you can turn this into a `<a>` element (styled as a button) that opens in a new tab, like the one in `ScheduleNewAppointment`. It shouldn't need a loading state or anything.

Reply:
Went with Narin's suggestion.